service/src/test/resources/gpc/output/ExpectedEhrDocument.xml

```diff
@@ -0,0 +1,54 @@
+<COPC_IN000001UK01 xmlns="urn:hl7-org:v3">
+    <id root="test-message-id"/>
+    <creationTime value="20200101010101"/>
+    <versionCode code="V3NPfIT3.0"/>
+    <interactionId root="2.16.840.1.113883.2.1.3.2.4.12" extension="COPC_IN000001UK01"/>
+    <processingCode code="P"/>
+    <processingModeCode code="T"/>
+    <acceptAckCode code="NE"/>
+    <communicationFunctionRcv>
+        <device>
+            <id root="1.2.826.0.1285.0.2.0.107" extension=""/>
```

Comment:
Refer to story - this should not be blank

Reply:
Sorry, forgot to set the value in the testing class.